tools/scripts/db-drop.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-docker exec -it tno-database bash -c "psql -U \$POSTGRES_USER \$POSTGRES_DB -c 'DROP SCHEMA public CASCADE; CREATE SCHEMA public;'"
+docker exec -i tno-database bash -c "psql -U \$POSTGRES_USER \$POSTGRES_DB -c 'DROP SCHEMA public CASCADE; CREATE SCHEMA public;'"
```

Comment:
Is the `-it` required?

Reply:
I believe this is the "t" I had to put back in to get db-refresh to work. This is another one of those last-minute fixes that was lost late yesterday afternoon.

Reply:
The "t" causes a teletype error when I run db-drop.sh.